src/lib/annotations/Annotator.js

```diff
@@ -21,6 +17,8 @@ import {
     SELECTOR_ANNOTATION_BUTTON_DRAW_CANCEL,
     SELECTOR_ANNOTATION_BUTTON_DRAW_ENTER,
     SELECTOR_ANNOTATION_BUTTON_DRAW_POST,
+    SELECTOR_BOX_PREVIEW_BTN_ANNOTATE_POINT,
```

Comment:
Ideally I don't want preview specific button css to exist in annotations. I will look into ways to make that configurable and passed in through annotationOptions or something